ci_scripts/setup-keyring-servers.sh

```diff
@@ -18,8 +18,5 @@ pykmip-server -f "$SCRIPT_DIR/../contrib/pg_tde/pykmip-server.conf" -l /tmp/kmip
 TV=$(mktemp)
 { exec >$TV; vault server -dev; } &
 sleep 10
-export ROOT_TOKEN=$(cat $TV | grep "Root Token" | cut -d ":" -f 2 | xargs echo -n)
-echo "export ROOT_TOKEN=$ROOT_TOKEN"
-if [ -v GITHUB_ACTIONS ]; then
-    echo "ROOT_TOKEN=$ROOT_TOKEN" >> $GITHUB_ENV
-fi
+echo $(cat $TV | grep "Root Token" | cut -d ":" -f 2 | xargs echo -n) > /tmp/vault_test_token
+echo "write vault token to /tmp/vault_test_token"
```

Comment:
Not sure I am the biggest fan of hardcoding this path. Wouldn't it be cleaner to have `VAULT_TOKEN_PATH` which points to a temporary file generated with mktemp? Of course that still does not solve the issue of the hardcoded port so maybe pointless.